src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+
+    <div class="calendar page__calendar calendar--start-day--sun">
```

Comment:
add modifier calendar__day--31, 31 means how many days in the calendar should be visible

Task: `Add a modifier month-length for the calendar with values 28, 29, 30 and 31`